docs/source/guide/python_api.rst

```diff
@@ -56,8 +56,43 @@ You can create a project for a dataset given the dataset ID returned from the up
     >>> studio.create_project(dataset_id, project_name, modality)
     <project ID>
 
+Get Cleanset ID
+---------------
+You can get the ID of the latest cleanset for your project given the project ID returned from the create project step.
```

Comment:
Should state "why" the user is getting their cleanset ID (e.g it's needed to export results)